### PR TITLE
2015 to 2016 and remove extra tab

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Copyright from 2015 to 2016 and remove extra tab i manifest.

### CODE REVIEW

File: src/administrator/manifests/packages/pkg_weblinks.xml [hunk 1]
```diff
@@ -4,14 +4,14 @@
 	<packagename>weblinks</packagename>
 	<creationDate>December 2012</creationDate>
 	<packager>Joomla! Project</packager>
-	<copyright>(C) 2005 - 2015 Open Source Matters. All rights reserved.</copyright>
+	<copyright>(C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
```
Comment (yvesh) [inline src/administrator/manifests/packages/pkg_weblinks.xml]: With the latest JoRobo version you can use ##YEAR## here, then we never have to update it again..

File: src/administrator/manifests/packages/pkg_weblinks.xml [hunk 2]
```diff
@@ -4,14 +4,14 @@
 	<packagename>weblinks</packagename>
 	<creationDate>December 2012</creationDate>
```
Comment (yvesh) [inline src/administrator/manifests/packages/pkg_weblinks.xml]: ##DATE## here

File: src/administrator/manifests/packages/pkg_weblinks.xml [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
 	<packager>Joomla! Project</packager>
-	<copyright>(C) 2005 - 2015 Open Source Matters. All rights reserved.</copyright>
+	<copyright>(C) 2005 - 2016 Open Source Matters. All rights reserved.</copyright>
 	<packageremail>admin@joomla.org</packageremail>
 	<packagerurl>www.joomla.org</packagerurl>
 	<author>Joomla! Project</author>
 	<authorEmail>admin@joomla.org</authorEmail>
 	<authorUrl>www.joomla.org</authorUrl>
 	<version>3.4.1</version>
```
Comment (yvesh) [inline src/administrator/manifests/packages/pkg_weblinks.xml]: ##VERSION## here

**Comment by andrepereiradasilva** [comment]
@yvesh done

**Comment by andrepereiradasilva** [comment]
ok fixed sorry

**Comment by yvesh** [comment]
@Test working fine.. Thanks

**Comment by Kubik-Rubik** [comment]
Thank you, @andrepereiradasilva!

**Pull request merged**